GUIDE_CREATE_CERTIFICATE.md

```diff
@@ -0,0 +1,48 @@
+# How to create a new certificate?
+
+This guide explains how to create a new self signed certifcate, used for SORMAS to SORMAS communication or 
+for accessing the SurvNet interface.
+
+**Important:** This is the first version of this guide. Please get in contact if anything is not clear or 
+you have suggestions on how to improve the guide, the source code or the underlying architecture.
+   
+### Using the certificate generation script
+
+1. **Important**: If you customized the install paths during server installation, please open ``generate-cert.sh`` in a text editor and 
+   set the ``ROOT_PREFIX`` variable to the same value which was used during the installation. <br/>
+   Run ``bash ./generate-cert.sh``
+2. For the generation of the certificate, the following data is needed: a password, a *Common Name* (CN) 
+    and an *Organization* (O). These may be set in environment variables (recommended), or provided 
+    manually as the script executes.
+    * The password environment variable should be named ``SORMAS_S2S_CERT_PASS``.
+    * the *Common Name* environment variable should be named ``SORMAS_S2S_CERT_CN``.<br/>
+    **Important**: for Germany, this value should be the SurvNet Code Site. <br/>
+    E.g. *2.03.1.01.*
+    * the *Organization* (O) environment variable should be named ``SORMAS_S2S_CERT_ORG``.<br/>
+    **Important**: for Germany, this value should be the name of the Health Department (Gesundheitsamt) 
+    to which the SORMAS instance will be assigned. <br/>
+    E.g. *GA Braunschweig*
+3. After providing the requested data, the certificate files will be generated. <br/>
+   The generated certificate has a validity of 3 years. 
+   The certificate files will be available in the root SORMAS directory, in the folder ``/sormas2sormas``.
+4. The generated ``.p12`` file should not be shared with third parties. <br/>
+   The generated ``.crt`` file will be verified and shared with other SORMAS instances, from which this instance
+   will be able to request data. Conversely, in order to enable other SORMAS instances to request data from this 
+   instance, their certificate files should be obtained and added to the local truststore. More details can be found
+   in the next section.
+5. If the ``SORMAS_PROPERTIES`` environment variable is available, the relevant properties will be 
+    automatically set by the script.
+    * If the ``SORMAS_PROPERTIES`` environment variable is not available, the properties which need to be added will be 
```

Comment:
`Else` instead of `If the ``SORMAS_PROPERTIES`` environment variable is not available,`?